pkg/controller/service/service_controller_test.go

```diff
@@ -618,6 +631,10 @@ func TestProcessServiceCreateOrUpdate(t *testing.T) {
 
 }
 
+func getExpectedKey(svc *v1.Service) string {
```

Comment:
cache.MetaNamespaceKeyFunc does the job.

Reply:
fixed.